### PR TITLE
268 divisionbyzeroerror

### PR DESCRIPTION
**Pull request by mschwoer**
- fix division by zero error
- give some hint in case a bruker DDA file is opened
- fix issues with E2E cache cleanup

### CODE REVIEW

File: alphadia/data/bruker.py [hunk 1]
```diff
@@ -65,13 +65,20 @@ def __init__(
...
-                    )
-                    raise ValueError(
-                        "Unexpected cycle shape. Will only retain first frame group"
+                        "Could not find cycle shape. Please check if this is a valid DIA data set."
                     )
+                    raise e
+                else:
+                    if cycle_shape != 1:
+                        msg = (
+                            "Unexpected cycle shape. Will only retain first frame group"
```
Comment (mschwoer) [inline alphadia/data/bruker.py:78]: @GeorgWa The error message sounds like processing will continue, but apparently it does not .. shall we rephrase it?

Comment (GeorgWa) [reply alphadia/data/bruker.py:78]: Yes, let's rephrase this to make it clear that we fail here.

**Review by GeorgWa — APPROVED** [review]
LGTM
